org.garudalinux.firedragon.appdata.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.garudalinux.firedragon</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0 AND LicenseRef-proprietary=https://raw.githubusercontent.com/Floorp-Projects/Floorp-private-components/main/LICENSE</project_license>
+  <name>FireDragon</name>
+  <summary><![CDATA[Floorp fork build using custom branding & settings]]></summary>
+  <developer_name>FireDragon Team</developer_name>
+  <icon type="stock">org.garudalinux.firedragon</icon>
+  <description>
+    <p><![CDATA[FireDragon is a browser based on the excellent Floorp browser. It was customized to have dr460nized-fitting aestetics as well as many opiniated settings by default. As this browser was originally a Librewolf fork, we are trying to integrate its best patches and tweaks in the new base. The most important features in addition of Floorps own ones can be found below.]]></p>
+    <ul>
+      <li><![CDATA[Searx & Whoogle search engines added, with the possibility to run locally]]></li>
+      <li><![CDATA[The default search engine is Garuda's Searx instance]]></li>
+      <li><![CDATA[Dark Reader]]></li>
+      <li><![CDATA[Sweet theme added]]></li>
+      <li><![CDATA[Custom, dr460nized branding 🐉]]></li>
+      <li><![CDATA[Keeping settings in sync with Librewolfs changes]]></li>
+      <li><![CDATA[Firefox accounts are enabled using a custom self-hosted sync server (ffsync.garudalinux.org)]]></li>
+      <li><![CDATA[Faster webpages loading from: Custom Firedragon settings, Inclusion of FastFox tweaks]]></li>
+      <li><![CDATA[Artwork by @SGS]]></li>
+      <li><![CDATA[Hidden Navigation buttons instead of being greyed out when they are inactive]]></li>
+      <li><![CDATA[Spoofing Chrome on Windows (through Floorp's User Agent switcher) when not using Fingerprinting]]></li>
+      <li><![CDATA[PBMode Security]]></li>
+    </ul>
+  </description>
+  <branding>
+    <color type="primary" scheme_preference="light">#d281ca</color>
+    <color type="primary" scheme_preference="dark">#d281ca</color>
+  </branding>
+  <launchable type="desktop-id">org.garudalinux.firedragon.desktop</launchable>
+  <categories>
+    <category>Network</category>
+    <category>WebBrowser</category>
+  </categories>
+  <keywords>
+    <keyword>garudalinux</keyword>
+    <keyword>web</keyword>
+    <keyword>browser</keyword>
+    <keyword>internet</keyword>
+  </keywords>
+  <url type="homepage">https://firedragon.garudalinux.org/</url>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Home</caption>
+      <image>https://gitlab.com/garuda-linux/firedragon/settings/-/raw/master/home.png</image>
```

Comment:
don't link to images on a branch. pick a commit or tag. otherwise it will break at one point.

same below

Reply:
Fixed in 290d79ddedc0a8834dc3110e8269fcd8a7c5d84a.